source/backend/api/Models/Concepts/Form/FileFormModel.cs

```diff
@@ -0,0 +1,13 @@
+using Pims.Api.Models.Lookup;
+
+namespace Pims.Api.Models.Concepts
+{
+    public class FileFormModel : BaseAppModel
+    {
+        public long Id { get; set; }
+
+        public long FileId { get; set; }
+
+        public LookupModel<string> FormTypeCode { get; set; }
```

Comment:
Not sure the form type matches the lookup exactly given it also has a document id

Reply:
Oh, right. Its a lookup with that one extra attribute. All right if I fix that in my next PR? will probably be easier.

Reply:
I will change it once it I merge it.